packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -20,6 +20,10 @@ enum TimelineEventType {
 class TimelineData {
   TimelineData({this.cpuThreadId, this.gpuThreadId});
 
+  /// Threshold for determining whether we will log a warning for a duplicate
```

Comment:
nit: threshold --> threshold in nanoseconds

Reply:
I think it's in microseconds?